apps/aecore/src/aec_tx_pool.erl

```diff
@@ -317,6 +342,10 @@ handle_cast_(Msg, State) ->
 handle_info(Msg, St) ->
     ?TC(handle_info_(Msg, St), Msg).
 
+handle_info_(origins_cache_gc, #state{} = State) ->
+    ok = origins_cache_gc(State),
+    erlang:send_after(30000, self(), origins_cache_gc),
```

Comment:
Same as above. Add an offset since it doesn't make sense to run this too often.

Reply:
This is to be executed every 30 seconds, which looked reasonable to me, but I can increase the delay... I don't have strong opinions here.

Reply:
Sorry, I read in too much from the previous code snippet. Disregard this.